### DISCUSSION TITLE
addNodes / addEdges batch update

### DISCUSSION THREAD

**Discussion by gkorland** [reaviz/reagraph#174] [Ideas]
I couldn't find a way to add abatch of nodes or edges.
I only found an API to add a single node or edge.

**Comment by amcdnl** [comment]
The system should just handle it automatically. React handles the change detection and resolution from the nodes array.

**Reply by gkorland** [reply]
Which array? do you mean the the `GraphNode[]` and `GraphEdge[]` I pass as an argument to the component?

**Reply by amcdnl** [reply]
Yes sir
